src/site/stages/build/process-cms-exports/transformers/node-event_listing.js

```diff
@@ -36,13 +36,17 @@ const transform = (entity, { ancestors }) => ({
   fieldDescription: getDrupalValue(entity.fieldDescription),
   fieldIntroText: getDrupalValue(entity.fieldIntroText),
   fieldMetaTitle: getDrupalValue(entity.fieldMetaTitle),
-  fieldOffice:
-    entity.fieldOffice[0] &&
-    !ancestors.find(r => r.entity.uuid === entity.fieldOffice[0].uuid)
-      ? {
-          entity: entity.fieldOffice[0],
+  /* eslint-disable no-nested-ternary */
```

Comment:
I wonder if there is a cleaner way to write this?

Reply:
Any thoughts?